tests/unit/test_auto_deploy_task.py

```diff
@@ -99,6 +101,7 @@ def test_client(portainer_service_mock: aioresponses, client: TestClient):
 
 
 async def test_wait_for_dependencies_no_portainer_up(client: TestClient):
+    assert client.app  # nosec
```

Comment:
What does `#nosec` mean here?

Reply:
so this is called [defensive programming](https://swcarpentry.github.io/python-novice-inflammation/10-defensive/index.html), got that from @pcrespov .
Also in VSCode, you have probably the pylance extension, that used to show some red lines in the line 105 because app could be None.
First with the assertion I ensure it is not None, it is also what is expected. The ```#nosec``` shows that I do know that the assertion will be removed in non-debug code (pylance will also issue a warning, that I have code checks in a assertion). the ```#nosec ``` removes the warning.

Reply:
thx